src/file_pipe_log/pipe.rs

```diff
@@ -83,15 +83,16 @@ impl<F: FileSystem> SinglePipe<F> {
     ) -> Result<Self> {
         let paths = vec![Path::new(&cfg.dir).to_path_buf()];
         let alignment = || {
-            fail_point!("pipe::force_set_alignment", |_| { 16 });
+            fail_point!("file_pipe_log::open::force_set_alignment", |_| { 16 });
             0
         };
         let default_format = LogFileFormat::new(cfg.format_version, alignment());
 
         // Open or create active file.
-        if active_files.is_empty() {
+        let no_active_files = active_files.is_empty();
+        if no_active_files {
             let path_id = DEFAULT_PATH_ID;
-            let file_id = FileId::new(queue, FileSeq::default());
+            let file_id = FileId::new(queue, 1 as FileSeq);
```

Comment:
What's the issue with 0?

Reply:
At the previous work, we build the prerequisite that **All file_seqno starts from 1** and keep this regulation in later developments.
For example:
https://github.com/tikv/raft-engine/blob/8dd2a39f359ff16f5295f35343f626e0c10132fa/src/file_pipe_log/pipe.rs#L271-L272
IMO, I still recommend to keep it.

Reply:
So to put it plainly, the issue with 0 is it will break backward compatibility (for debug binary). You should replace 1 with a constant.